camptocamp/mel-dataplatform
# [Search page] : Pagination

### Description

This PR adds a pagination component to the search page.

### To test

With the local dump, you will need to decrease the pageSize in SearchResultsComponent to 2-3 in order to see the pagination work properly. It is set to 18 because the ticket asked for a display of 3x6 cards.
